layers/+readers/pdf/packages.el

```diff
@@ -131,8 +131,13 @@
         "q"              'tablist-quit
         "g"              'pdf-occur-revert-buffer-with-args
         "r"              'pdf-occur-revert-buffer-with-args
+        "C-j"            'next-error
```

Comment:
What does "error" mean in this context?

Reply:
Ah, I thought that I had removed these bindings. Well, I am not sure exactly what it means, but what is does is jump to next line and 'follow`. The name of the command is very misleading (you can read in the docstring of `next-error-follow-minor-mode`, that the 'error' functionality also works for occur and diff mode).
I will probably suggest to the Emacs developers to create a more useful alias for it (in Emacs).

So the `'next-error-follow-minor-mode` is useful, because it does not select the pdf-buffer when using it. `next/previous` error will select the pdf-buffer when using them, so I don't find them very useful for `occur-mode`. I will remove them (which I had wanted to do already)